src/sdk-reference/cpp/1/search-result/index.md

```diff
@@ -1,22 +1,6 @@
 ---
 layout: sdk.html.hbs
 title: SearchResult
-description: How to deal with Kuzzle search results
+description: SearchResult class documentation
 order: 400
 ---
```

Comment:
This file was never displayed. The content is in `cpp/1/search-result/introduction/index.md`